COMMERCE-12950 - Commerce Product description is displayed with HTML Tags in Paragraph Fragment

victorcdp opened the pull request:
Related Issue: [COMMERCE-12950](https://liferay.atlassian.net/browse/COMMERCE-12950)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

noornajj commented:
ci:test:sf

noornajj commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-portal-source-format/6253/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-portal-source-format/builds/6253/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d157dfa16ee97ddd0425114fbc41f621b01cea40">d157dfa16ee97ddd0425114fbc41f621b01cea40</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-128446">LPS-128446</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/4fb28db41dbb8400748e13449251c00bf306b386">4fb28db41dbb8400748e13449251c00bf306b386</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-source-format/6253/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-source-format/builds/6253/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-22.liferay.com/job/test-portal-source-format/6253/">test-portal-source-format#6253</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-source-format/builds/6253/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/125">brianikim#125</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2002.aspx">/Liferay DXP 7.4/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/97269.aspx">master - noornajj &gt; brianikim - PR#125 - 2021-03-02[12:46:30]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/16948/">publish-spira-report#16948</a><br />

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/5545/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5545/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 23 out of 23 jobs passed in 4 hours 44 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d157dfa16ee97ddd0425114fbc41f621b01cea40">d157dfa16ee97ddd0425114fbc41f621b01cea40</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d157dfa16ee97ddd0425114fbc41f621b01cea40">d157dfa16ee97ddd0425114fbc41f621b01cea40</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1598">Acceptance Upstream DXP (master) #1598</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406300/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/351093/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/464074/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406307/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406301/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/454858/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435948/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435947/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406304/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 23 out of 23 jobs PASSED</strong></summary><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/5545/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/416222/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/416221/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406300/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435949/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/416223/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/351093/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/464074/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406308/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406305/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406302/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2785/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406307/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406303/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406306/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406301/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435946/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/454858/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435948/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/219276/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/435947/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406299/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406304/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5545/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/5545/">test-portal-acceptance-pullrequest(master)#5545</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5545/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/125">brianikim#125</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/2004.aspx">/Liferay DXP 7.4/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/97573.aspx">master - noornajj &gt; brianikim - PR#125 - 2021-03-02[15:56:16]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/17007/">publish-spira-report#17007</a><br />

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/787

evanilson-santana committed:
LPS-194538 Change relationship from objects for some tests that need to see the filter section in the aggregation field

Commit:
LPS-198057 Uses a new WebKeys.AUI_SCRIPT_DATA on each validation. We need to do it because in ScriptTag we are appending all js code inside <aui:script>, in the same WebKeys.AUI_SCRIPT_DATA.

So, if we have <aui:script> code in fragments we are going to add it to the same WebKeys.AUI_SCRIPT_DATA and it is going to be rendered in the page, so we need to avoid it since it can cause js problems.

In our particular case we have the rich text input that includes @liferay_editor["editor"], at the end it calls ckeditor.jsp and it has <aui:script>. It caused the following error when the page is loaded, since the ckEditor is not in the page but the js yes:

(index):3300 Uncaught TypeError: Cannot read properties of null (reading 'setHTML')
    at createEditor ((index):3300:20)

brookedalton committed:
LPS-198917 Hide fragment duplicates that are added to the draft page

AliciaGarciaGarcia committed:
LPS-201893 use the new tag on the fragment

gislayne-vitorino committed:
LPS-198234 (dynamic-data-mapping-test) Add test for DDMFormInstanceTableReferenceDefinition

evanilson-santana committed:
LPS-194538 Remove the IsEqualToIsAvailableForLongIntegerFields since the case is being covered by CanCreateEntryWithAggregationField

gislayne-vitorino committed:
LPS-198234 (dynamic-data-mapping-test) Add test for DDMFormInstanceVersionTableReferenceDefinition

evanilson-santana committed:
LPS-194538 Improvement in the test CanCreateEntryWithAggregationField

evanilson-santana committed:
LPS-194538 Remove the IsEqualToIsAvailableForIntegerFields since the case is being covered by CanCreateEntryWithAggregationField

evanilson-santana committed:
LPS-194538 Add task to add entries to ensure the aggregation field is working

evanilson-santana committed:
LPS-194538 Improvement of the test RangeIsAvailableForDateFields to add entries and ensure the aggregation is working

ctang3 committed:
LPS-199203 Fix inability to change the locale of the preview render when reviewing a Layout change with multiple locales

smottal committed:
COMMERCE-12433 Semantic Versioning

smottal committed:
COMMERCE-12433 Autogenerated Code (buildREST)

smottal committed:
COMMERCE-12433 Add module dependencies

smottal committed:
COMMERCE-12433 User View and Edit panel

smottal committed:
COMMERCE-12433 User image and language (yaml)

smottal committed:
COMMERCE-12433 Update API

gislayne-vitorino committed:
LPS-198232 (dynamic-data-mapping-test) Add test for DDMFormInstanceRecordVersionTableReferenceDefinition

smottal committed:
COMMERCE-12433 User fullname definition API (yaml)

evanilson-santana committed:
LPS-194538 Add variables for selectTitleField macro

jaimelr10 committed:
LPS-199304 Add logic to validate and create an endpoint

jaimelr10 committed:
LPS-199304 Add needed string utils

jaimelr10 committed:
LPS-199304 Add types

jaimelr10 committed:
LPS-199304 Change styles to match design

jaimelr10 committed:
LPS-199304 Extract endpoint configuration fields to components

jaimelr10 committed:
LPS-199304 Add component for path parameter configuration handling

jaimelr10 committed:
LPS-199304 Add path parameter input

jaimelr10 committed:
LPS-199304 Add retrieve type dropdown

Youness-ELMO committed:
LPS-197809 - Accessibility: Data Set Fragment

Youness-ELMO committed:
LPS-197809 - fix source format issue

Commit:
LPS-201024 Force it to use our patched 1.2.LIFERAY-PATCHED-2 version to use log4j2. commons-logging:1.2 used log4j1.

The following dependency chain of compileIncludePlatform:

+--- com.google.cloud:google-cloud-translate -> 2.6.0
|    +--- commons-logging:commons-logging:1.2

Commit:
LPS-201024 Avoid importing log4j 1 package. It's imported by the compile-included grpc-netty-shaded-1.50.2.jar, which includes netty-common-4.1.77, which in turn supports Log4J 2.

gislayne-vitorino committed:
LPS-201074 (translation-test) Add test for TranslationEntry

Commit:
LPS-197934 Refactor UpgradeSetResultsSetTotalMethodCheck class for attend the setResultsAndTotal method with two parameters

Commit:
LPS-197934 Refactor tests UpgradeSetResultsSetTotalMethodCheck class

Commit:
LPS-197934 Documentation

gislayne-vitorino committed:
LPS-201074 (translation-service) Add AssetEntry and JournalArticle as children of TranslationEntry to avoid leftover data when discarding TranslationEntry

david-truong committed:
LPS-199203 don't update the language for user when previewing

danvb112 committed:
LPS-201391 Add new styles to Model Builder Minimap

danvb112 committed:
LPS-201391 Add styles to React flow control buttons in Model Builder

Commit:
LRQA-83496 Add some asserts to CanCP20BeCreated Test

Commit:
LRQA-83496 Delete AccountUsersAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete ObjectsAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete RemoteAppsAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete TestsAccountsAreCreatedForCP20 Test

jaimelr10 committed:
LPS-199304 Add dependency to package.json

Commit:
LPS-199195 portal-search-solr8-impl: Bump apache zookeeper

https://liferay.atlassian.net/browse/LPS-199195

Commit:
LRQA-83496 SF

jaimelr10 committed:
LPS-199304 Add frontend validation on path parameter property

riccardo-alberti committed:
COMMERCE-12878 Add null checks

sergiojimcos committed:
LPS-201192 Add test cases for object entry

Commit:
LPS-200192 Getting RepositoryProvider via Snapshot, prepare for next

Commit:
LPS-200192 Move RepositoryProviderImpl into module and make it a component

jaimelr10 committed:
LPS-199304 buildLang

Commit:
LPS-200192 Move LiferayFileEntrySocialActivityManager,LiferayFileShortcutSocialActivityManager and  LiferayFolderSocialActivityManager into module

Commit:
LPS-200192 Semantic versioning

sergiojimcos committed:
LPS-201192 Update Object Definition export CSV test cases

baharturk committed:
COMMERCE-12433 Add new locators

baharturk committed:
COMMERCE-12433 Add new macros

baharturk committed:
COMMERCE-12433 Add new functional tests

AliciaGarciaGarcia committed:
LPS-201893 remove the .lfrbuild-releng-ignore to let the saved content tag be used by other modules without failure.

sergiojimcos committed:
LPS-201192 Renamse object_entry.csv file and adapt _prepareCSVFile for Object Definitions and Object Entries

sergiojimcos committed:
LPS-201192 Add test case for export Objec Entry in CSV from other site

sergiojimcos committed:
LPS-201192 Refactor export Object Entry test cases

Commit:
LPS-201288 Replace blocking ServiceProxyFactory with Snapshot

JanisZhang committed:
LPS-199426 Replace non-blocking ServiceProxyFactory usages by using Snapshot in PortletFileRepositoryUtil

JanisZhang committed:
LPS-199426 Postpone PortalInstanceRegistered process of WorkflowMetricsPortalInstanceLifecycleListener by making it extending InitialRequestPortalInstanceLifecycleListener to avoid NPE caused by PortletFileRepositoryUtil.fetchPortletRepository while deleteBackgroundTask during startup.

sergiojimcos committed:
LPS-201192 Extract common code and set SCOPE_KEY parameter to be checked

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 Fix typo

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 SF

Commit:
LPS-196765 Fix mistake

CrescenzoRegaLF committed:
LPS-195448 Fix bypass related to redirect

brittneyq committed:
LRCI-3899 Account for repository name parameter when retrieving release repository

Commit:
LPS-196765 Revert

This reverts commit b45fd4c2a3803bff1965f0a7e5958c6a665c0f4a.

michaelhashimoto committed:
LRCI-3890 Shift build parameters into the build object definition

michaelhashimoto committed:
LRCI-3890 Add parameters to the build information page

michaelhashimoto committed:
LRCI-3890 Remove all references to the build entity parameters

CAustin582 committed:
LPS-202078 Remove unused service

michaelhashimoto committed:
LRCI-3890 Fixes after deployment to LXC

michaelhashimoto committed:
LRCI-3890 Move the build parameters into its own panel

pei-jung committed:
LPS-198917 Hide FragmentEntryLinks if its layout is hidden

shuyangzhou committed:
LPS-202327 Disable eclipse.bundle.setTCCL

Commit:
LPS-202092 Replace non-blocking ServiceProxyFactory with Snapshot

jkappler committed:
LPS-202106 Don't use ddmStructureId parameter and create a new one, so that it doesn't overlap with the filtering one

qz1010 committed:
LPS-202266 avoid error change year

jkappler committed:
LPS-202106 If we are in a highlighted structure, we want to show all of the items, not limiting it by folder

jkappler committed:
LPS-202106 It makes no sense to show the filtering of the structure when we are already in a highlighted structure

adolfopa committed:
LPS-182875 Remove feature flag from functional tests

adolfopa committed:
LPS-182875 Remove feature flag from portal.properties

adolfopa committed:
LPS-182875 Remove feature flag references from code

jkappler committed:
LPS-202106 Don't depend on the configuration when selecting structures

Commit:
LPS-196765 Revert since it's a private host

Commit:
LPS-196765 Revert to use PG access

This reverts commit 437dfe90c61ea0f5fde5a7c3fa381b86147b7620.

Commit:
LPS-196765 Revert to use PG access

This reverts commit a22ed154

lfbesada committed:
LPS-202222 Disallows to convert to page template for no-content type layouts

AkramYoussoufi committed:
LPS-201679 Adding 3 tests in AllowUsersToEditSchemaProperties testcase for APIBuilder Component

AliciaGarciaGarcia committed:
LPS-201893 SF

smailbiala committed:
LPS-201678 Verify error message and non-saving behavior for Filters exceeding 1000 characters

Luizcarlosqueiroz committed:
LPS-201555 Fix notifications type path

Luizcarlosqueiroz committed:
LPS-201555 Make notification checkbox path more resilient

Commit:
LPS-196765 Revert

Commit:
LPS-196765 Revert To use PG access

This reverts commit 70bea3db64c05131d4a832ca6d1c9cd6bd4d8c21.

Commit:
LPS-202145 Add the right value to be checked

pedroamarall committed:
LPS-201672 | Update createAPISchema.macro

pedroamarall committed:
LPS-201672 | add test's from APIApplicationSchema

shuyangzhou committed:
LPS-202327 Switch classloader for ES due its SPI usage.

ling-alan-huang committed:
LPS-202266 Move logic

interaminense committed:
LRAC-15075 Follow up unit tests

fmarcio committed:
LRAC-15075 update snapshots

michaelhashimoto committed:
LRCI-3890 SF

michaelhashimoto committed:
LRCI-3902 Create a Jethr0 Message Listener that will send updates to Jethr0BuildUpdater

allen-nagtalon committed:
LPS-201273 Update macro to search for partial text

allen-nagtalon committed:
LPS-201273 Add test to view related entry details of publication changes

Commit:
LPS-202221 update maven-war-plugin version

JoyceWang08211 committed:
LRQA-84480 Add test

victorg1991 committed:
LPS-193840 Not needed form here

This is causing a redirect that is trying to load a url that doesn't exist and throw an error in console

AliciaGarciaGarcia committed:
LPS-201021 check the permissions for the structureModelResourceName

Commit:
LPS-202221 update css builder for module

Commit:
LPS-202221 restore css builder for theme template

Commit:
LPS-202221 update service builder to 1.0.469 for maven

marcogalluzzi committed:
LPS-202242 Make trashHelper available to all portlets

DiegoHu97 committed:
LPS-202085 Convert Bytes into MegaBytes

sandrodw3 committed:
LPS-202218 Disable warning if it doesn't have type

ealonso committed:
LPS-202249 Discard code under LPS-149256

Mikellorza committed:
LPS-202248 Delete unused finder

Mikellorza committed:
LPS-202248 buildService. Delete unused finder.

# breaking_change_report

## What portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagUtil and portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagPersistence

portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagUtil.java and portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagPersistenc.java. Now there are not include the methods findByG_LikeN,findByG_N_First,findByG_N_Last,findByG_N_PrevAndNext,removeByG_N and countByG_N

## Why

These changes are because of there are not use in the portal.

----

victorg1991 committed:
LPS-197787 Check if the component is mounted

victorg1991 committed:
LPS-197787 Add tests for RulesModal

Mikellorza committed:
LPS-202248 Restore case-sensitive=false in order to provide the same behavior for all databases. Then in the AssetTagLocalServiceImpl and AssetTagServiceImpl should be taken into account the case-sensitive where we have to take account case-sensitive

Mikellorza committed:
LPS-202248 buildService

lfbesada committed:
LPS-202220 Adds a specific success message when the content type is properly changed

lfbesada committed:
LPS-202220 Adds language key

lfbesada committed:
LPS-202220 BuildLang

ealonso committed:
LPS-202257 Sets default value for _fragmentElementId to avoid NPE, like we do in DefaultFragmentRendererContext

ealonso committed:
LPS-202257 No needed since we already have a default value

ling-alan-huang committed:
LPS-202266 SF

ling-alan-huang committed:
LPS-202266 Auto fix year

DiegoHu97 committed:
LPS-202264 Remove not used file

4lejandrito committed:
LPS-201192 Simplifying

4lejandrito committed:
LPS-201192 Renaming method

4lejandrito committed:
LPS-201192 Working with strings instead of streams

rafaellajordao committed:
LPS-196141 add ConfigureTimeWithoutAutosaveUnderInstanceSettings test

4lejandrito committed:
LPS-201192 As used

rafaellajordao committed:
LPS-196141 add ViewAccessibilityOfLockedPagesUnderInstanceSettings test

Commit:
LPS-198962 Decrease validation delay

22LucaGiannotta22 committed:
LPS-195448 Fix functional test

rafaellajordao committed:
LPS-196141 add EnableAllowAutomaticUnlockingProcessUnderInstanceSettingsViaKeyboard test

rafaellajordao committed:
LPS-201194 add DROPDOWN_MENU_ITEM_ACTIVE path

rafaellajordao committed:
LPS-201194 add OrderByTitleInSearchResults test

rafaellajordao committed:
LPS-201194 add OrderByDisplayDateInNewSearchResults test

4lejandrito committed:
LPS-201192 No need to use a temp file

rafaellajordao committed:
LPS-201194 add ViewOrderButtonDisabledWhenFilterByRecent test

Commit:
LPS-196765 Revert

rafaellajordao committed:
LPS-201194 modify LIST_ENTRY_X path

rafaellajordao committed:
LPS-201194 add RevertSortDirectionOfSearchResultsViaKeyboard test

Mikellorza committed:
LPS-202248 Make finder case-insensitive

pyoo47 committed:
LRCI-3888 Increase number of threads

brittneyq committed:
LRCI-3899 Use generic tomcat dir for release

Mikellorza committed:
LPS-202275 Delete unused methods

Mikellorza committed:
LPS-202275 Delete unused methods

# breaking_change_report

## What portal-kernel/src/com/liferay/asset/kernel/service/AssetTagLocalService.java and portal-kernel/src/com/liferay/asset/kernel/service/AssetTagLocalServiceUtil.java

Delete getSocialActivityCounterOffsetTags and getSocialActivityCounterPeriodTags methods.

## Why

These methods are not used.

----

Mikellorza committed:
LPS-202275 Delete unused method

Mikellorza committed:
LPS-202275 Delete unused methods

# breaking_change_report

## What portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagFinder.java and portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagFinderUtil.java

Delete findByG_N_S_E method.

## Why

This method is not used.

----

adolfopa committed:
LPS-182431 Remove mvc path from URL

KevinQueiroz committed:
LRQA-82281 Fix campaignName propertie to goToMDF macro

brianchandotcom committed:
LRCI-3902 prep next

brianchandotcom committed:
LPS-200552 prep next

brianchandotcom committed:
LPS-200552 prep next

pei-jung committed:
LPS-201074 Add or update TranslationEntry updates the JournalArticle as part of the workflow, but we should not define JournalArticle as the child because TranslationEntry doesn't own it.

kevenleone committed:
ISSD-1746 Use Loading, SWR, Simplify

brianchandotcom committed:
LRCI-3888 prep next

magjed4289 committed:
LPS-202151 Test-fix for CanDownloadFiledWithAllFieldsSelected

petershin committed:
Revert "LPS-87890 drop hours, minutes, and seconds"

This reverts commit d089b938a9abc63c6250ee533b94b3e540e300b5.

Commit:
ISSD-1367 Fix CI Fail

petershin committed:
LPS-87890 Human readability

Commit:
LPS-179185 remove implements in WorkflowDefinitionResourceImpl

ambrinchaudhary committed:
LPS-193840 Update tests

ambrinchaudhary committed:
LPS-193840 If button is disabled it won't be clicked

pei-jung committed:
LPS-201074 Add TranslationEntry as a child of JournalArticle so it will be removed when JournalArticle is removed

Commit:
LPS-202184 Add property to skip virtual instance and organize the testcase

pei-jung committed:
LPS-201074 Sort

pei-jung committed:
LPS-201074 JournalArticle is updated when TranslationEntry is added. Return the JournalArticle so all related entities can be cleaned up properly.

michaelhashimoto committed:
LRCI-3907 Add logic that will check for jars if no lpkg files exist

Commit:
LPS-202184 Fix title field the CanViewTypeOnFieldEntry

pyoo47 committed:
LRCI-3888 Default timeout should be in seconds

Commit:
LPS-202258 Fix testcase with wrong label

pyoo47 committed:
LRCI-3888 Fix getSubmittedCount()

pyoo47 committed:
LRCI-3888 Add getSubmittedTaskCount()

brianchandotcom committed:
LPS-87890 prep next

qz1010 committed:
LPS-202266 performance

drewbrokke committed:
LPS-202307 gradle-plugins-workspace: adds client-extension.schema.json

brianchandotcom committed:
LRCI-3907 prep next

JanisZhang committed:
LPS-200601 Fix test

Commit:
LPS-196141 Remove redundant step

Commit:
LPS-196141 Trigger validation by clicking outside of field instead of saving

brianchandotcom committed:
LPS-202307 Wordsmith

Commit:
LPS-196141 Sort

Commit:
LPS-196141 Use correct elements

brianchandotcom committed:
LPS-202221 prep next

Commit:
LPS-201194 Refactor locator for filter and order

Commit:
LPS-201194 Migrate case and use correct elements

Commit:
LPS-201194 Migrate and refactor cases

qz1010 committed:
LPS-201623 Auto SF

Hijky committed:
LPS-202151 Update a testcase to remove duplicate method

Commit:
LPS-202314 Modify case to make it stable

Commit:
LPS-201912 Update the tests for the preview URL

ealonso committed:
LPS-202337 Removes LPS-165346 Feature Flag

victorg1991 committed:
LPS-202326 Update test

Mikellorza committed:
LPS-202275 Use name without any transformation because now the finders are case-insensitive

Commit:
LPS-196367 Fix typo

brianchandotcom committed:
LPS-201912 SF

brianchandotcom committed:
LRCI-3888 prep next

brianchandotcom committed:
LRCI-3899 prep next

brianchandotcom committed:
LPS-196141  SF

natocesarrego committed:
LPS-197934 Little change in description

natocesarrego committed:
LPS-197934 Update documentation

jaimelr10 committed:
LPS-199304 Add logic to validate and update an endpoint

brianchandotcom committed:
LPS-201893 Add .releng now that ignore is removed

brianchandotcom committed:
LPS-201893 Rename isViewMode to setViewMode

brianchandotcom committed:
LPS-201893 This is not ContentRatingsFragmentRenderer.java

brianchandotcom committed:
LPS-201893 Rename

brianchandotcom committed:
COMMERCE-12433 Sort

joelrocha68 committed:
LPS-185475 Fix test

brianchandotcom committed:
Add u103

brianchandotcom committed:
LPS-201192 Rename/SF

Commit:
LPS-202100 Fix typo

brianchandotcom committed:
LPS-201192 SF

brianchandotcom committed:
LPS-201192 SF

Commit:
LPS-202100 Add case for LPS-187651

carolmariaabb committed:
LPS-202304 Use ID instead of name because the self many-to-many relationship has the same name for both child and parent

shuyangzhou committed:
LPS-202295 Turn on logging so that we can see what is failing.

shuyangzhou committed:
LPS-202295 Switch context classloader, this is neeeded because JDK11 does not bundle jaxb libs, we have to load it from shielded container classloader.

brianchandotcom committed:
LPS-199304 Wordsmith

brianchandotcom committed:
LPS-199304 Regen

brianchandotcom committed:
portal-impl 94.0.1 prep next

brianchandotcom committed:
object-web 1.0.189 prep next

Commit:
COMMERCE-12950 Product Description Info Field Type should be HTML so it renders correctly on Fragments

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/091180b85b554b0233b4d5c6358d1513f5c5fc78">091180b85b554b0233b4d5c6358d1513f5c5fc78</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-12950">COMMERCE-12950</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/23329cf3db6e18a195c7bdd0981ac92237428ec0">23329cf3db6e18a195c7bdd0981ac92237428ec0</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-source-format/6898/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-source-format/builds/6898/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-16.liferay.com/job/test-portal-source-format/6898/">test-portal-source-format#6898</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-source-format/builds/6898/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/125">brianikim#125</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2463668508">[master] ci:test:sf - victorcdp &gt; brianikim - PR#125 - 2023-11-22[11:01:37]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-16.liferay.com/job/publish-testray-report/3645/">publish-testray-report#3645</a><br/></div></div>
